src/odemis/gui/cont/tabs.py

```diff
@@ -639,6 +629,16 @@ def get_display_priority(cls, main_data):
         else:
             return None
 
+    @call_in_wx_main
+    def display_acquired_data(self, data):
+        # get the top right view port
+        views = self.tab_data_model.views.value[1]
+        for s in views.getStreams():
+            views.removeStream(s)
+        streams = data_to_static_streams(data)
+        for s in streams:
+            views.addStream(s)
```

Comment:
It doesn't work this way. Because, the function "addStream" of the StreamBarController class adds the passed stream to the .streams VA. This leads to adding every stream twice in the checklist. I looked at the implementation of the function addStream , and reused only the line of code that adds the steam to the ACQUIRED panel, which is creating ab object of StreamController. So like this:
`stream_cont = StreamController(self._panel.pnl_cryosecom_acquired, s, self._tab_data_model,
                                           show_panel=True, view=view, sb_ctrl=self._overview_stream_controller)
stream_cont.stream_panel.collapse(True)`

Reply:
I now understand. Thanks for the explanation. As Bassim said, it's better to handle it by distinguishing StaticStreams from the "live" streams in .streams. Then you can call .addStream() normally. 

Concerning the behaviour of the view, for now, let's keep it simple: every new stream is added to the view and shown. 
Later, part of the "feature bookmark" work by Bassim, the behaviour will be fine-tuned: the view will only show the streams acquired at the current location. From the point of view of the acquisition manager, that doesn't change anything, so you don't need to worry.